doc/nrf/gs_assistant.rst

```diff
@@ -0,0 +1,85 @@
+.. _gs_assistant:
+
+.. |GSA| replace:: Getting Started Assistant
+
+.. gs_assistant_intro
+
+Installing the |NCS| with |GSA|
+###############################
+
+The |GSA| application provides you with step-by-step instructions for installing the toolchain for |NCS|.
+
+The application allows you to check which of the required tools are already installed on your computer and verify the installation.
+It also guides you through installing the parts of the toolchain that you are missing.
+In addition, it will help you clone the |NCS| repositories and set up your first project in |SES|.
+
+.. gs_assistant_installing-gsa
+
+Installing the |GSA|
+********************
+
+The |GSA| app is available in `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS.
+
+To install the Assistant app, complete the following steps:
+
+.. _assistant_setup:
+
+1. `Download nRF Connect for Desktop`_ for your operating system.
+#. Install and run the tool on your machine.
+#. In the APPS section, click :guilabel:`Install` next to |GSA|.
+
+.. figure:: images/gs-assistant_installation.PNG
+   :alt: The nRF Connect for Desktop main window
+   
+   The nRF Connect for Desktop main window
+
+The app is installed on your machine, and the :guilabel:`Install` button changes to :guilabel:`Open`.
+
+.. gs_assistant_installing-ncs
+
+Installing the |NCS|
+********************
+
+Once you installed |GSA|, open it in nRF Connect for Desktop.
+The |GSA| app will open with the Overview section displayed in the main window.
+
+.. figure:: images/gs-assistant_overview.PNG
```

Comment:
I don't think this screenshot is needed (people will see it when they open the app).